appium/webdriver/extensions/remote_fs.py

```diff
@@ -74,7 +74,7 @@ def push_file(self, destination_path: str,
                 with open(source_path, 'rb') as f:
                     file_data = f.read()
             except IOError:
-                message = 'source_path {} could not be found. Are you sure the file exists?'.format(source_path)
+                message = f'source_path {source_path} could not be found. Are you sure the file exists?'
```

Comment:
I'd prefer to quote the actual values to properly read them in case they contain spaces